packages/core/content-manager/server/routes/admin.js

```diff
@@ -182,6 +182,21 @@ module.exports = {
         ],
       },
     },
+    {
+      method: 'GET',
+      path: '/single-types/:model/actions/numberOfDraftRelations',
```

Comment:
should we add the new routes in [`Relations in main view`](https://www.notion.so/strapi/Relations-in-main-view-Architecture-c977fb72e36942a68be56d08c274bff5) notion page?